isminet/models/validators.py

```
@@ -22,20 +22,66 @@ def validate_mac(v: Optional[str]) -> Optional[str]:
             )
-        validated.append(mac.lower())
-    return validated
+
+        return validated
+
+    return validate_list
+
+
+# Create validators using the factory
+validate_ip_list = create_list_validator(
+    IPv4Address, "invalid_ip", "Invalid IPv4 addresses: {values}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)